### ISSUE TITLE
Delta on every apply

### ISSUE THREAD

**Issue by alex-quach-arc-boats** [airplanedev/terraform-aws-airplane-agents#19]
Hey there! Whenever I run `tf apply`, it always wants to update this:

```
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.airplane_agent.aws_iam_role.agent_execution_role will be updated in-place
  ~ resource "aws_iam_role" "agent_execution_role" {
      ~ assume_role_policy    = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Action    = "sts:AssumeRole" -> [
                          + "sts:AssumeRole",
                        ]
                        # (2 unchanged elements hidden)
                    },
                ]
              - Version   = "2008-10-17" -> null
            }
        )
        id                    = "terraform-20220823183333352700000005"
        name                  = "terraform-20220823183333352700000005"
        tags                  = {
            "Project" = "internal_tools"
        }
        # (9 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

  # module.airplane_agent.aws_iam_role.agent_role will be updated in-place
  ~ resource "aws_iam_role" "agent_role" {
      ~ assume_role_policy    = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Action    = "sts:AssumeRole" -> [
                          + "sts:AssumeRole",
                        ]
                        # (2 unchanged elements hidden)
                    },
                ]
              - Version   = "2008-10-17" -> null
            }
        )
        id                    = "terraform-20220823183333670100000006"
        name                  = "terraform-20220823183333670100000006"
        tags                  = {
            "Project" = "internal_tools"
        }
        # (9 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

  # module.airplane_agent.aws_iam_role.default_run_role will be updated in-place
  ~ resource "aws_iam_role" "default_run_role" {
      ~ assume_role_policy    = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Action    = "sts:AssumeRole" -> [
                          + "sts:AssumeRole",
                        ]
                        # (2 unchanged elements hidden)
                    },
                ]
              - Version   = "2008-10-17" -> null
            }
        )
        id                    = "terraform-20220823183334044300000007"
        name                  = "terraform-20220823183334044300000007"
        tags                  = {
            "Project" = "internal_tools"
        }
        # (9 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 3 to change, 0 to destroy.
╷
│ Warning: Argument is deprecated
│ 
│   with module.airplane_agent.aws_ecs_cluster.cluster,
│   on .terraform/modules/airplane_agent/main.tf line 7, in resource "aws_ecs_cluster" "cluster":
│    7:   capacity_providers = ["FARGATE"]
│ 
│ Use the aws_ecs_cluster_capacity_providers resource instead
│ 
│ (and one more similar warning elsewhere)
```

And there's a deprecation warning too. Seems like adding the Version to policies could help. Thanks!

```
Terraform v1.3.0
on linux_amd64
+ provider registry.terraform.io/hashicorp/archive v2.2.0
+ provider registry.terraform.io/hashicorp/aws v4.9.0
+ provider registry.terraform.io/hashicorp/dns v3.2.3
+ provider registry.terraform.io/hashicorp/local v2.2.2
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.1.2
+ provider registry.terraform.io/hashicorp/template v2.2.0
```

**Comment by joshma** [comment]
Hi @alex-quach-arc-boats , thanks for filing these issues. We'll take a look and get back to you soon.

**Comment by andy-hubble** [comment]
+1 for this :)

**Comment by zhan-airplane** [comment]
Hi @alex-quach-arc-boats @andy-hubble we've merged a change in v0.3.6 to fix these issues, could you give it another try? Thanks for the bug report!

**Comment by alex-quach-arc-boats** [comment]
> No changes. Your infrastructure matches the configuration.
> 
> Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are needed.

:ok_hand:  thanks for the quick turnaround!
